src/dotnet/APIView/APIView/APIView.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
Seems like this repo is not setup to build `netcoreapp2.2` projects, switch TargetFramework to netcoreapp2.1 here and in tests